packages/console-reporter/src/stage/crew/console-reporter/ConsoleReporter.ts

```diff
@@ -30,10 +30,12 @@ import {
     ProblemIndication,
 } from '@serenity-js/core/lib/model';
 import { Instance as ChalkInstance } from 'chalk'; // eslint-disable-line unicorn/import-style
+import type { FileSystem } from 'packages/core/lib/io';
```

Comment:
You'll want `@serenity-js/core/lib/io` instead

Reply:
yeah, I noticed vs code is not doing great with this repo 😄

Reply:
I wonder what causes it 🤔